spec/overview.html

```diff
@@ -1,28 +1,28 @@
 <emu-clause id="overview">
   <h1>Overview</h1>
 
-  <p>This section contains a non-normative overview of the ECMAScript 2025 Internationalization API Specification.</p>
+  <p>This section contains a non-normative overview of the ECMAScript  Internationalization API Specification.</p>
```

Comment:
Has it disappeared?? 👀

```suggestion
  <p>This section contains a non-normative overview of the ECMAScript 2026  Internationalization API Specification.</p>
```